swagger-spec/institutions/registrations_list.yaml

```diff
@@ -1 +1,156 @@
 # /institutions/{institution_id}/registrations/
+get:
+  summary: List all institution affiliated registrations
+  description: >-
+    This is an object representing registrations affiliated with an institution. 
+    You can retrieve it to see a paginated list of all registrations affiliated with an institution.
+
+    ##Query Params
+
+    ###Page: `page=<Int>`
+
+    Page number of results to view. Defaults to page 1.
+
+    ###Field Name: `filter[<fieldname>]=<Str>`
+
+    Fields and values to filter the search results on.
+
+    ###Filterable Fields: `id`, `title`, `description`, `public`, `tags`, `category`, `date_created`, `date_modified`, `root`, `parent`, `contributors`, `preprint`
+
+    Fieldnames that Institutions can be filtered based on.
+
+  parameters:
+    - description: ''
```

Comment:
These parameters can be removed, but there should be an `institution_id` parameter added.